core/src/main/java/com/radixdlt/p2p/capability/AppVersionCapability.java

```diff
@@ -62,25 +62,28 @@
  * permissions under this License.
  */
 
-package com.radixdlt.modules;
+package com.radixdlt.p2p.capability;
 
-import com.google.inject.AbstractModule;
-import com.google.inject.Provides;
-import com.google.inject.Singleton;
-import com.radixdlt.p2p.capability.Capabilities;
-import com.radixdlt.p2p.capability.LedgerSyncCapability;
+import com.radixdlt.monitoring.ApplicationVersion;
+import java.util.Map;
 
-public class CapabilitiesModule extends AbstractModule {
+/** A "capability" used for sharing application version */
+public record AppVersionCapability(ApplicationVersion applicationVersion) {
+  public static final String NAME = "app-version";
+  public static final String CONFIG_STRING = "string";
```

Comment:
Would prefer this is `CONFIG_VERSION` and `"version"` I think, as "string" feels a little unclear to me